package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "personalizedcontent",
-  "version": "2.2.1",
+  "version": "1.2.3",
```

Comment:
This is actually bumped by [`npm version`](https://docs.npmjs.com/cli/version) and shouldn't be modified by hand. Running `npm version` executes a number of lifecycle scripts and produces a lightweight tag necessary for release tracking.